src/main/java/pages/ContactUsPage.java

```diff
@@ -12,6 +12,8 @@
     private int timeOut = 5;
 
     private By contactUsPageTitle = By.xpath("//h1[contains(text(),'Contact')]");
+    //Added by FvZ
```

Comment:
Het is niet nodig om dergelijke comments toe te voegen aangezien git dit netjes voor ons bij houdt.